arch/arm/src/stm32f7/stm32_spi.c

```diff
@@ -2177,6 +2177,7 @@ FAR struct spi_dev_s *stm32_spibus_initialize(int bus)
   else
 #endif
     {
+      leave_critical_section(flags);
```

Comment:
@bkueng Why not just delete the return NULL?  As FAR struct stm32_spidev_s *priv = NULL;  if unassigned but a port.

Reply:
Good point. I updated accordingly.